rllib/env/wrappers/open_spiel.py

```diff
@@ -1,9 +1,12 @@
-from gymnasium.spaces import Box, Discrete
-import numpy as np
-import pyspiel
 from typing import Optional
 
+import numpy as np
```

Comment:
Just curious, is this a new sorting convention? Three blocks: a) included python modules, b) other third-party imports, c) ray imports?

Reply:
The three categories you describe (and the formating changes here) conform with PEP-8 I belive.
-> https://peps.python.org/pep-0008/#imports
I think we should adhere at least to PEP-8. The google style guide is optional.

I use PyCharm's "optimize imports" feature to do these, but PyCharm also groups "import x as y" vs "import x" and separates them which makes for apparent missing alphabetical order.